src/anonymous-credentials/presentation-builder.ts

```diff
@@ -242,11 +242,13 @@ export class PresentationBuilder extends Versioned {
   enforceBounds(
     credIdx: number,
     attributeName: string,
-    min: number,
-    max: number,
+    vmin: number | Date,
```

Comment:
You are expecting `number`, `Date` and `string` in encoder so the same type should be used here.
And type would be changed in `BlindedCredentialRequestBuilder.enforceBounds` as well.

Reply:
can you elaborate on `BlindedCredentialRequestBuilder`? what do you mean by the type will be changed?

> You are expecting number, Date and string in encoder so the same type should be used here.

but this conflicts with this feedback:

> The conversion of date to integer should be done with the encoder and not duplicated here.

Reply:
ignore previous comment about "BlindedCredentialRequestBuilder.enforceBounds" (deleted it) - i misread your comment. i can update this once we address the feedback for this method

Reply:
> can you elaborate on BlindedCredentialRequestBuilder? what do you mean by the type will be changed?

Check the mentioned function. It's enforcing bounds as well.

> but this conflicts with this feedback:

I am talking about the type of function arg. In the encoder, you can/could pass the date as a `number|string|Date` but this expects only `number|Date`. Its inconsistent.

Reply:
>Check the mentioned function. It's enforcing bounds as well.

yeah i misread that in my original comment and amended it, assumnig you are replying to comments via email/app or similar and didnt see the update

> I am talking about the type of function arg. In the encoder, you can/could pass the date as a number|string|Date but this expects only number|Date. Its inconsistent.

im aware thats the suggestion here but maybe i was misunderstanding your point in the other thread, initially i thought you were saying *not* to do conversion in that method at all, but i think your point now is rather about sharing logic than actually calling the encoder method

Reply:
> yeah i misread that in my original comment and amended it, assumnig you are replying to comments via email/app or similar and didnt see the update

Sorry. Was replying through browse but didn't see that.

> your point now is rather about sharing logic than actually calling the encoder method

Yes.

Reply:
got it now after resolving other thread, will update